definitions/example.js

```diff
@@ -0,0 +1,16 @@
+const bigQueryLogs = require("../");
+
+const bigQueryLogsModels = bigQueryLogs({
+  logsSchema: "stackdriver_logs",
+  defaultConfig: {
+    tags: ["bq_audit_logs"],
+    type: "view"
+  },
+});
+
+// Override the sessions and user table type to "table".
+bigQueryLogsModels.bigQueryComputeLogs.type("table").config({
```

Comment:
Just FYI, I wouldn't make this a table in the example even. The table for us us like 16GB, and given it's a simple transformation without joins, I can't see any reason why this should be materialized.